clients/go/pkg/commands/createInstance_command_test.go

```diff
@@ -372,7 +373,7 @@ func TestCreateWorkflowInstanceWithResultCommand(t *testing.T) {
 		Request: &pb.CreateWorkflowInstanceRequest{
 			WorkflowKey: 123,
 		},
-		RequestTimeout: utils.DefaultTestTimeoutInMs,
+		RequestTimeout: longPollMillis,
```

Comment:
Is `longPollMillis` also coming from `activateJobs_command_test.go`? In this case, shouldn't we use `utils.DefaultTestTimeout` instead of `5*time.Second` when creating the context here as well?

Reply:
Yes, that's a mistake